tests/assets/selenium-grid/README.md

```diff
@@ -1,4 +1,3 @@
 Download locations:
 - https://github.com/SeleniumHQ/selenium/releases/download/selenium-3.141.59/selenium-server-standalone-3.141.59.jar
```

Comment:
lets remove this file, its stale anyways. We have the download links now in the code.